sys/net/link_layer/l2util/l2util.c

```diff
@@ -76,6 +93,14 @@ int l2util_eui64_from_addr(int dev_type, const uint8_t *addr, size_t addr_len,
                 return -EINVAL;
             }
 #endif  /* defined(MODULE_CC110X) || defined(MODULE_NRFMIN) */
+#if defined(MODULE_NRF24L01P)
+        case NETDEV_TYPE_NRF24L01P:
+            if (addr_len >= 3 && addr_len <= 5) {
```

Comment:
The code would be a bit easier to read, if both address length checks would be identical. (See below, where the check is in different order: `addr_len <= 5 && addr_len >= 3`)

Reply:
I thought the same when I was squashing, but must have forgotten to change that.